js/script.js

```diff
@@ -0,0 +1,20 @@
+let mobileMenu = document.getElementById('hamburger-div');
+let mobileMenuCancel = document.getElementById('cancel-hamburger');
+let mobileMenuCancel1 = document.getElementById('cancel-butt1');
+let mobileMenuCancel2 = document.getElementById('cancel-butt2');
+let mobileMenuCancel3 = document.getElementById('cancel-butt3');
+mobileMenu.addEventListener('click', () => {
+  document.querySelector('.mobile-div').classList.add('show');
+});
+mobileMenuCancel.addEventListener('click', () => {
+  document.querySelector('.mobile-div').classList.remove('show');
+});
+mobileMenuCancel1.addEventListener('click', () => {
+  document.querySelector('.mobile-div').classList.remove('show');
+});
+mobileMenuCancel2.addEventListener('click', () => {
+  document.querySelector('.mobile-div').classList.remove('show');
+});
+mobileMenuCancel3.addEventListener('click', () => {
+  document.querySelector('.mobile-div').classList.remove('show');
+});
```

Comment:
Too many repetitions here. We should always make our code DRY. The best method here would be to create a function and reuse it where necessary

```js
const removeActiveClass = () => {
  navUl.classList.remove('active');
}

mobileMenuCancel.addEventListener('click', removeActiveClass);
mobileMenuCancel1.addEventListener('click', removeActiveClass);
mobileMenuCancel2.addEventListener('click', removeActiveClass);
mobileMenuCancel3.addEventListener('click',removeActiveClass);
```
This way our function becomes more reusable and our code snippet become shorter and more maintainable.